packages/frontend-api/src/Model/Resolver/Order/OrderItemResolverMap.php

```diff
@@ -36,6 +41,19 @@ protected function map(): array
                 'vatRate' => function (OrderItem $orderItem) {
                     return $orderItem->getVatPercent();
                 },
+                'productMainImage' => function (OrderItem $orderItem) {
```

Comment:
for a change, this will "conflict" with https://github.com/shopsys/shopsys/pull/3164/commits/3dc77392437312370527edac16d11afa797b6e3a - whole product was added to `OrderItemDecorator.types` there :smile: It will be hopefully merged soon so you can update your branch accordingly then

Reply:
It should be sufficient to just drop this [order item schema product main image](https://github.com/shopsys/shopsys/pull/3362/commits/a532182d221ce2b70d5d7a9aef32543976d24ae0) commit and regenerate the schema. I'll do this after the mentioned commit is merged.